docs/provision-api-reference.md

```diff
@@ -114,37 +114,10 @@ removed for this user.
-| ROAMING                        | Calls                                                            | N/A                                                   |
-| ROAMING_DATA                   | Data access while roaming                                        | N/A                                                   |
-| SMS_IN                         | Incoming SMS                                                     | N/A                                                   | Y
-| SMS_OUT                        | Outgoing domestic SMS                                            | N/A                                                   | Y
-| SMS_OUT_INTER                  | International SMS                                                | N/A                                                   | Y
-| SMS_PREMIUM                    | Premium content SMS                                              | N/A                                                   |
-| VOICE_CALLFORWARD              | Forward calls                                                    | 1 or more "condition" : "fwd_number" with condition in  unconditional \| busy \| no_reply \| unreachable
-| VOICE_CALLWAITING              | If busy, let new callers wait instead of dropping with busy tone | N/A                                                   |
-| VOICE_IN                       | Incoming calls                                                   | N/A                                                   | Y
-| VOICE_MISSED_CALL_ALERT        | Missed call alert                                                | N/A                                                   | Y
-| VOICE_OUT                      | Outgoing domestic calls                                          | N/A                                                   | Y
-| VOICE_OUT_INTER                | International calls                                              | N/A                                                   |
-| VOICE_PREMIUM                  | Premium content calls                                            | N/A                                                   |
-| VOICE_VOICEMAIL                | Voicemail service                                                | N/A                                                   | Y
-| VOICE_VOLTE                    | Voice calls over LTE                                             | N/A                                                   |
-| VOICE_VOWIFI                   | Voice calls over WiFi                                            | N/A                                                   |
-| VOICEMAIL_DISABLE_NOTIFICATION | Disable voicemail notifications                                  | N/A                                                   |
-
 **Error cases:**
 * msisdn unknown
 * service name unknown
+
+### Service list
```

Comment:
Should this be h3 (is it a child of `## UPDATE`? Or h2?
```suggestion
## Service list
```

Reply:
It actually is a child of update, as that's the only call in the provisioning API v1 that uses this list.